core/tauri/scripts/ipc-protocol.js

```diff
@@ -0,0 +1,52 @@
+// Copyright 2019-2023 Tauri Programme within The Commons Conservancy
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-License-Identifier: MIT
+
+(function () {
```

Comment:
probably want to open with a semicolon here
```suggestion
;(function () {
```